### PR TITLE
[Term Entry] C# Math Functions .Sqrt() #1613

### PR DESCRIPTION
**Pull request by cwhaley98**
### Description

As part of the Introduction to Open Source to learning GIt & GitHub, I have contributed to adding a new term entry describing how the square root method works while using C#. I have also provided examples using different integers returning with their respective square roots.

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [ x] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            {
+                double x = Math.Sqrt(144);
+
+                Console.WriteLine(x);
+            }    
+        }
+    }
+```
+
+This example results in the following output:
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:45]: I would suggest having only one example for this method. There isn't really any additional functionality that is being demonstrated by having both examples. I would say the second is more complete, so, you may want to use that one in place of the first and delete the codebyte.

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+The `.Sqrt()` function will can take on one parameter at a time. As shown above _x_ is the value we want to find the sqaure root of.
+
+## Example
+
+The following example will implement how `Math.Sqrt()` will take its given value in the function and return its square root by print it with the `Console.WriteLine()` method.
+
+```cs
+using System;
+
+namespace MySqaureRoot
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:31]:
```suggestion
namespace MySquareRoot
```

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+The **`Math.Sqrt()`** method can return the sqaure root of a specified number. If the value of the number happens to be equal to Nan, NegativeInfinity, or PositiveInfinity, that value is returned.
+
+## Syntax
+
+```pseudo
+public static double Sqrt(double x);
+```
+
+The `.Sqrt()` function will can take on one parameter at a time. As shown above _x_ is the value we want to find the sqaure root of.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:22]:
```suggestion
The method takes only one parameter, the variable `x`, of type `double`. The function will return a positive value of type `double` unless the value passed is one of the following:
-If `x` is negative, the function will return `NaN` (not a number).
-If `x` is `NaN`, the function will return `NaN`.
-If `x` is `PositiveInfinity`, the function will return `PositiveInfinity`.
```

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+---
+Title: '.Sqrt()'
+Description: 'Returns the square root of a number.'
+Subjects:
+    - 'Computer Science'
+    - 'Code Foundations'
+Tags:
+    - 'Sqrt'
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:8]:
```suggestion
    - 'Arithmetic'
    - 'Numbers'
    - 'Methods'   
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cwhaley98 First review is complete. Just a handful of edits to better align this entry with our standards.

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md (hunk 1 above)
Comment (cwhaley98) [reply content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md]: Thank you for the suggestions! I went ahead and deleted the codebyte and replaced the first example with the second one. Please let me know if there is anything else I need to add on here!

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+```
+
+The method takes only one parameter, the variable `x`, of type `double`. The function will return a positive value of type `double` unless the value passed is one of the following:
+-If `x` is negative, the function will return `NaN` (not a number).
+-If `x` is `NaN`, the function will return `NaN`.
+-If `x` is `PositiveInfinity`, the function will return `PositiveInfinity`.
+
+## Example
+
+The following example will implement how `Math.Sqrt()` will take its given value in the function and return its square root by print it with the `Console.WriteLine()` method.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:31]:
```suggestion
The following example demonstrates the `Math.Sqrt()` method. Four different values are passed to method, and the return values are printed with the `Console.WriteLine()` method.
```

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    - 'learn-c-sharp'
+    - 'paths/computer-science'
+---
+
+The **`Math.Sqrt()`** method returns the square root of the number given.
+
+## Syntax
+
+```pseudo
+public static double Sqrt(double x);
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:21]:
```suggestion
Math.Sqrt(x);
```

**Comment by cwhaley98** [comment]
Hey @caupolicandiaz I've completed the suggested changes, and just waiting for another review. 👍 Please let me know if there is anything else I need to do as this is my first time using GitHub and creating a pull request. Thanks!

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cwhaley98 A couple of additional edits to wrap up this review. Thanks

File: content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+```
+
+The method takes only one parameter, the variable `x`, of type `double`. The function will return a positive value of type `double` unless the value passed is one of the following:
+-If `x` is negative, the function will return `NaN` (not a number).
+-If `x` is `NaN`, the function will return `NaN`.
+-If `x` is `PositiveInfinity`, the function will return `PositiveInfinity`.
+
+## Example
+
+The following example demonstrates the `Math.Sqrt()` method. Four different values are passed to method, and the return values are printed with the `Console.WriteLine()` method.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/sqrt/sqrt.md:31]:
```suggestion
The following example demonstrates the `Math.Sqrt()` method. Four different values are passed to the method, and the return values are printed with the `Console.WriteLine()` method.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cwhaley98 one small correction, otherwise this looks good

**Comment by cwhaley98** [comment]
@caupolicandiaz Thank you for all the help! I've committed the last one you suggested to the file 👍 Please let me know if there's anything else I need to do 😁

**Review by caupolicandiaz — APPROVED** [review]
@cwhaley98 LGTM. This is ready for the next review

**Review by SSwiniarski — APPROVED** [review]
@cwhaley98 Looks good to merge to me.

**Comment by SSwiniarski** [comment]
@cwhaley98 Thanks for the solid contribution! The entry can be seen live at: https://www.codecademy.com/resources/docs/c-sharp/math-functions/sqrt
image

**Pull request merged**